src/resources/js/v2/tickets-commerce.js

```diff
@@ -180,6 +183,28 @@ tribe.tickets.commerce = {};
 		$document.trigger( 'afterSetup.tecTicketsCommerce', [ $container ] );
 	};
 
+	/**
+	 * Get purchaser data if available.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container Container for the purchaser info fields.
+	 *
+	 * @return {Object}
+	 */
+	obj.getPurchaserData = function ( $container ) {
```

Comment:
Use ES6 here. `() => {}`

Reply:
I just kept it consistent with the other methods of the file, but if we are only converting the newer methods to ES6 then  we can do this.

Reply:
We should move anything we have the opportunity to be ES6.